### ISSUE TITLE
MacOS Binary link broken on v.1.9.4 release page

### ISSUE THREAD

**Issue by handsomelatino** [cryptoadvance/specter-desktop#1689]
The link appears broken:
https://github.com/cryptoadvance/specter-desktop/releases/download/v1.9.4/SpecterDesktop-v1.9.4.dmg

Correct one seems to be (taken from specter.solutions page):
https://github.com/cryptoadvance/specter-desktop/releases/download/v1.9.4/Specter-v1.9.4.dmg

**Comment by k9ert** [comment]
Thank you, pretty good point. Fixed.

**Comment by handsomelatino** [comment]
You're welcome. I also noticed it broken in older release tags too, which isn't really important but just in case. Thanks! Specter rules 💪

**Comment by k9ert** [comment]
Yes, true. We changed the naming scheme and haven't adjusted the templating logic. Need to fix that ...
